### DISCUSSION TITLE
Products "lost" when editing default channel language and currency.

### DISCUSSION THREAD

**Discussion by agoransson** [vendurehq/vendure#1233] [Q&A] [answered]
I've been building a storefront connecting to Vendure and everything has been super-smooth so far. Until I today decided to change the default channel language and currency. Then all products were lost from search - they don't show in either my storefront query or in administrator interface. I can see them in the collections view in administrator view though, but they won't show up in any search.

I've tried to rebuild the search index but that doesn't help.

What would you recommend to get the products visible again? I'm working with the default data currently until the storefront is finished.

**Comment by michaelbromley** [comment, marked as answer]
Hi,

So when the demo data was first populated, the translatable parts (names, slugs, descriptions) were created only for the then-default language (English I am guessing). Now that you've changed the default language, the search indexer is looking for versions in that language to index, but they don't exist. Only in the old default language.

You can test this by adding some names & descriptions in the new default language to a couple of products & variants, and then reindexing, and seeing if they then show up in the search query.
